src/VestingWalletWithCliff.sol

```diff
@@ -40,48 +33,25 @@ abstract contract VestingWalletWithCliff is VestingWallet {
     }
 
     /**
-     * @dev Override of getter for the amount of releasable native assets to return 0 prior to meeting the cliff.
-     */
-    function releasable() public view virtual override returns (uint256) {
-        if (_isBeforeCliff()) {
-            return 0;
-        }
-        return super.releasable();
-    }
-
-    /**
-     * @dev Override of getter for the amount of releasable `token` tokens to return 0 prior to meeting the cliff.
-     * `token` should be the address of an IERC20 contract.
+     * @dev Override of VestingWallet's `_vestingSchedule` to enforce releasing nothing until the cliff has passed.
      */
-    function releasable(address token) public view virtual override returns (uint256) {
+    function _vestingSchedule(uint256 totalAllocation, uint64 timestamp)
+        internal
+        view
+        virtual
+        override
+        returns (uint256)
+    {
         if (_isBeforeCliff()) {
             return 0;
         }
-        return super.releasable(token);
-    }
-
-    /**
-     * @dev Release the native assets that have already vested if the cliff has been passed.
-     *
-     * Emits a {EtherReleased} event.
-     */
-    function release() public virtual override isAfterCliff {
-        super.release();
-    }
-
-    /**
-     * @dev Release the tokens that have already vested if the cliff has been passed.
-     *
-     * Emits a {ERC20Released} event.
-     */
-    function release(address token) public virtual override isAfterCliff {
-        super.release(token);
+        return super._vestingSchedule(totalAllocation, timestamp);
     }
 
     /**
      * @dev Returns `true` if the current time is before the cliff.
      */
     function _isBeforeCliff() internal view returns (bool) {
-        return block.timestamp < (start() + cliffDuration());
+        return uint64(block.timestamp) < (start() + cliffDuration());
     }
 }
```

Comment:
Now that we're overriding `_vestingSchedule`, this function is only called once. It's also only one line, so we could simply remove it and replace the invocation with that line.

Reply:
done